docs/data-cookbooks/error-handling.md

```diff
@@ -21,6 +21,7 @@ Some best practices for error handling in Cumulus Workflows are:
 * States may be configured to `Retry` a task on specified failures to handle transient issues, such as those arising from resource allocation throttling, instead of failing the entire workflow. Cumulus supports the AWS retry configuration outlined [here](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-errors.html#amazon-states-language-retrying-after-error) and an example is provided in the `HelloWorld` step of the `RetryPassWorkflow` workflow defined in the Cumulus repository's [example](https://github.com/nasa/cumulus/blob/master/example/workflows.yml) `workflows.yml`.
 * Tasks downstream of failed tasks should understand how to pass along exceptions if required: If a task throws an error which is caught by the workflow configuration and passed to another state which also uses the CMA, the CMA overrides the exception key to `"None"` so the exception will not be passed to downstream tasks after the next state. This is okay if the exception is not needed in downstream tasks. However, `sf2snsEndLambdaFunction` does need an exception to understand the workflow is in a failed state, so the error should be re-thrown in any states between the original failed task and `sf2snsEndLambdaFunction`. In the example below, `CnmResponseFail` re-throws any errors passed by upstream tasks.
 * If multiple downstream tasks should run after a workflow task has thrown an error, for example sending a failure to a kinesis stream in addition to running the `sf2snsEndLambdaFunction`, this can handled by creating a second "failure" branch of the workflow.
+* Tasks that are lambdas should be configured to retry in the event of a Lambda Service Exception. See [this documentation](https://docs.aws.amazon.com/step-functions/latest/dg/bp-lambda-serviceexception.html) on configuring your workflow to handle transient lambda errors.
```

Comment:
:+1: good place to put this. I think it would also be helpful to throw a comment into the `CHANGELOG` pointing this suggestion out.